chart/values.yaml

```diff
@@ -705,7 +705,6 @@ config:
     statsd_host: '{{ printf "%s-statsd" .Release.Name }}'
   webserver:
     enable_proxy_fix: 'True'
-    expose_config: 'True'
```

Comment:
I simply remove this line, so the default value in `default_airflow.cfg` is used.